.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "**/charts/*"
+      - ".github/workflows/release.yml"
+      - "lib/*.js"
+      - "react/*.js"
+      - "routes/*.js"
+      - ".releaserc.yaml"
+  workflow_dispatch: {}
+
+concurrency:
+  group: release
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    outputs:
+      changed: ${{ steps.release.outputs.changed }}
+      previousVersion: ${{ steps.release.outputs.previousVersion }}
+      newVersion: ${{ steps.release.outputs.newVersion }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Release
+        id: release
+        uses: liatrio/github-actions/conventional-release@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
this token needs to belong to a user that is allowed to directly push to `main` and the `gh-pages` branch